management-gateway/src/main/java/org/frankframework/management/bus/ResourceNotFoundException.java

```diff
@@ -0,0 +1,46 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.management.bus;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.frankframework.core.IbisException;
+
+public class ResourceNotFoundException extends RuntimeException {
+	private static final Logger LOG = LogManager.getLogger(ResourceNotFoundException.class);
```

Comment:
Feel free to use the `@Log4J` annotation at the class level, to prevent this manual code.